<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>523</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>402</width>
    <height>523</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>402</width>
    <height>523</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Векторный калькулятор</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::DriveOptical"/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonStyle::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QGroupBox" name="ManagementBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Управление векторами</string>
    </property>
    <widget class="QPushButton" name="AddBtn">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>60</y>
       <width>90</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::ListAdd"/>
     </property>
    </widget>
    <widget class="QPushButton" name="RemoveBtn">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>90</y>
       <width>90</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::ListRemove"/>
     </property>
    </widget>
    <widget class="QPushButton" name="OpenBtn">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>120</y>
       <width>90</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Открыть</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
     </property>
    </widget>
    <widget class="QPushButton" name="SaveBtn">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>150</y>
       <width>90</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
     </property>
    </widget>
    <widget class="QListView" name="VectorView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>270</width>
       <height>220</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="VectorPreview">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>271</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="SaveChangesBtn">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>30</y>
       <width>90</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::MailMessageNew"/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="FuncTabBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>381</width>
      <height>161</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::TextElideMode::ElideNone</enum>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="CalcTab">
     <attribute name="title">
      <string notr="true">Калькулятор</string>
     </attribute>
     <widget class="QTextEdit" name="SentanceField">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="SolveBtn">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>71</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Решить</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="ResultField">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>361</width>
        <height>70</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="UtilityTab">
     <attribute name="title">
      <string>Разное</string>
     </attribute>
     <widget class="QComboBox" name="OperatorBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="OperandBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="ResultLine">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>361</width>
        <height>70</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ExecBtn">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Вычислить</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>402</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
